### ISSUE TITLE
Open passport from the Bot

### ISSUE THREAD

**Issue by grkamil** [TelegramMessenger/TGPassportJsSDK#2]
Is it possible to open Telegram Passport from the Bot?
I see that JS opens the link: tg://resolve?domain=telegrampassport.... , but if we send it through the Bot (sendMessage), Telegram doesnt recognize it as inline link.

**Comment by poulad** [comment]
@GrKamil No. It was possible in early days of tg passport but not anymore. Response from `@BotSupport` :

> I'm afraid that Telegram Passport URLs are disallowed to send inside inline buttons, because it is completely insecure, sorry.

**Comment by poulad** [comment]
I use this page in my tests to redirect user via browser:
<https://telegrambots.github.io/Telegram.Bot.Extensions.Passport/redirect.html>

**Comment by guardnow** [comment]
> I use this page in my tests to redirect user via browser:
> 
> <https://telegrambots.github.io/Telegram.Bot.Extensions.Passport/redirect.html>

**Comment by guardnow** [comment]
I had everything to telegraph please any
